### PR TITLE
Add dotColor and updated dots

### PR DESCRIPTION
**Pull request by TeaSeaLancs**
This PR updates the version of material-ui-dots and adds a dotColor property which passes through to the Dots component.

### CODE REVIEW

File: src/AutoRotatingCarousel.js [hunk 1]
```diff
@@ -195,6 +195,7 @@ export default class AutoRotatingCarousel extends Component {
                 index={modulo(this.state.slideIndex, this.props.children.length)}
                 style={landscape ? style.dotsLandscape : style.dots}
                 onDotClick={this.handleChange}
+                dotColor={this.props.dotColor}
```
Comment (leMaik) [inline src/AutoRotatingCarousel.js]: This prop should be destructured above.

**Comment by leMaik** [comment]
@TeaSeaLancs Thank you! :+1:

File: src/AutoRotatingCarousel.js (hunk 1 above)
Comment (TeaSeaLancs) [reply src/AutoRotatingCarousel.js]: Can you elaborate on what you mean by that? I'm not seeing any other destructuring of `this.props`.

Comment (leMaik) [reply src/AutoRotatingCarousel.js]: You're right… nevermind.

Comment (TeaSeaLancs) [reply src/AutoRotatingCarousel.js]: :+1:

**Pull request merged**